spec/actors/hyrax/actors/administrative_report_or_publication_actor_spec.rb

```diff
@@ -0,0 +1,9 @@
+# Generated via
+#  `rails generate hyrax:work AdministrativeReportOrPublication`
+require 'rails_helper'
+
+RSpec.describe Hyrax::Actors::AdministrativeReportOrPublicationActor do
```

Comment:
I think we should remove these since the actors are all just inheriting from base.. we don't need bogus specs hanging out doing nothing.